bin/pelican.auto.ezt

```diff
@@ -72,6 +72,24 @@ ASF_GENID = {
     'debug': [debug],
 }
 [end]
+
+[if-any uses_sitemap]
+SITEMAP = {
+    "exclude": [sitemap.exclude],
+    "format": "[sitemap.format]",
+    "priorities": {
+        "articles": [sitemap.priorities.articles],
+        "indexes": [sitemap.priorities.indexes],
+        "pages": [sitemap.priorities.pages]
```

Comment:
comma after each item is best-practice so that adding a item doesn't require modifying the prior line.